src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -29,15 +29,14 @@ import {
 
 // Styles
 import "./UserProfilePicture.scss";
-
-type UserProfilePictureProp = {
-  isAudioEffectDisabled?: boolean;
-  miniAvatars?: boolean;
-  avatarClassName?: string;
-  avatarStyle?: object;
-  containerStyle?: object;
-  setSelectedUserProfile: (user: WithId<User>) => void;
-  user: WithId<User>;
+import * as S from "./UserProfilePicture.styles";
+
+// This would be the global variables and configuration,
+// it is here only for the duration of the demo
+const DEMO_THEME = {
```

Comment:
just wondering, what's the plan for this?

Reply:
This will be removed in the next commit, it was here just as the name suggests, as a demo of how styled components would work.

The new implementation of the self-serve venue creation sets-up the theme provider in the index.tsx file so this it is redundant in this file